src/main/java/com/esri/samples/scene/feature_layer_extrusion/FeatureLayerExtrusionSample.java

```diff
@@ -0,0 +1,146 @@
+    Scene fxScene = new Scene(stackPane);
+    // for adding styling to any controls that are added 
+    fxScene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+    // set title, size, and add scene to stage
+    stage.setTitle("Feature Layer Extrusion Sample");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(fxScene);
+    stage.show();
+
+    // so scene can be visible within application
+    ArcGISScene scene = new ArcGISScene(Basemap.createTopographic());
+    sceneView = new SceneView();
+    sceneView.setArcGISScene(scene);
+    stackPane.getChildren().add(sceneView);
+
+    // get us census data as a service feature table
+    ServiceFeatureTable statesServiceFeatureTable = new ServiceFeatureTable("http://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer/3");
+    statesServiceFeatureTable.setFeatureRequestMode(ServiceFeatureTable.FeatureRequestMode.MANUAL_CACHE);
+
+    // load all features, with all fields (attributes), from service feature table
+    QueryParameters queryParams = new QueryParameters();
+    queryParams.setWhereClause("1=1");
+    statesServiceFeatureTable.populateFromServiceAsync(queryParams, true, Arrays.asList("*"));
+
+    // creates feature layer from table and add to scene
+    final FeatureLayer statesFeatureLayer = new FeatureLayer(statesServiceFeatureTable);
+    // feature layer must be rendered dynamically for extrusion to work
+    statesFeatureLayer.setRenderingMode(FeatureLayer.RenderingMode.DYNAMIC);
+    scene.getOperationalLayers().add(statesFeatureLayer);
+
+    // symbols are used to display features (US states) from table
+    SimpleLineSymbol lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFF000000, 1.0f);
+    SimpleFillSymbol fillSymbol = new SimpleFillSymbol(SimpleFillSymbol.Style.SOLID, 0xFF0000FF, lineSymbol);
+    final SimpleRenderer renderer = new SimpleRenderer(fillSymbol);
+    //  need to set an extrusion type, BASE HEIGHT extrudes each point from feature individually
+    renderer.getSceneProperties().setExtrusionMode(Renderer.SceneProperties.ExtrusionMode.BASE_HEIGHT);
+    statesFeatureLayer.setRenderer(renderer);
+
+    // set camera to focus on state features
+    Point lookAtPoint = new Point(-10974490, 4814376, 0, SpatialReferences.getWebMercator());
+    OrbitLocationCameraController orbitCamera = new OrbitLocationCameraController(lookAtPoint, 10000000);
+    sceneView.setCameraController(orbitCamera);
+
+    // create a control panel
+    VBox vBoxControl = new VBox();
+    vBoxControl.setMaxSize(200, 40);
+    vBoxControl.getStyleClass().add("panel-region");
+    stackPane.getChildren().add(vBoxControl);
+    StackPane.setAlignment(vBoxControl, Pos.TOP_LEFT);
+    StackPane.setMargin(vBoxControl, new Insets(10, 0, 0, 10));
+
+    // controls for extruding by total population or by population density
+    Button extrusionButton = new Button("Population Density");
```

Comment:
I prefer a `ToggleSwitch` here like we have on the feature layer rendering expressions samples.